0xProject/0x-monorepo
# Using timestamps, skip generation of already-up-to-date contract wrappers

## Description

Fix for #785 

## Testing instructions


Prepare: checkout 0x-monorepo, `cd packages/metacoin`, `yarn build`.
Then `yarn generate_contract_wrappers`, and ensure that the output says "Already up to date: ..." and not "Created: ...".

Also run new tests of abi-gen via `yarn test` in abi-gen folder.

## Types of changes




## Checklist:
